src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,89 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final String DATA_SPLITTER = " ";
+    private static final String COMMA_SPLITTER = ",";
+    private static final int NAME_INDEX = 0;
+    private static final int COUNT_INDEX = 1;
+    private static final String SEPARATOR = System.lineSeparator();
+
+    public void getStatistic(String fromFileName, String toFileName) throws RuntimeException {
```

Comment:
we dont need to declare throw of unchecked exceptions
```suggestion
    public void getStatistic(String fromFileName, String toFileName) {
```